utils/tracing/integ-tests/client-server/goclient/go.mod

```diff
@@ -2,13 +2,13 @@ module tests/tracing/python/client-server
 
 go 1.16
 
-replace go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.20.0
+replace github.com/ease-lab/vSwarm/utils/tracing/go => ../../../go
```

Comment:
ah here you did it but its a bit different then I expected..
What is the reason for the different paths